include/sensors/core/message_handlers/sensors.hpp

```diff
@@ -76,7 +76,15 @@ class SensorHandler {
                 break;
             }
             case can::ids::SensorType::pressure: {
-                client.send_pressure_sensor_queue(m);
+                switch (id) {
+                    case can::ids::SensorId::S1: {
```

Comment:
Is there protection to keep us from dereferencing a `null` queue pointer if there's only one pressure sensor on the pipette and a CAN message tries to specify the one that isn't configured? If not, we should have a NULL check in the `send_pressure_sensor_queue` functions.

Reply:
good call!